etn-ccis/blui-react-native-workflows
# Added auth context

Fixes # BLUI-5117



#### Changes proposed in this Pull Request:

- Added auth context, provider, hook
- Added test cases
- Added example



#### Screenshots / Screen Recording (if applicable)

- 
![Simulator Screenshot - iPhone 12 - 2024-03-13 at 16 03 40](https://github.com/etn-ccis/blui-react-native-workflows/assets/120575281/8b0baf3d-56f7-4c9d-9f78-a9b49eeaa6ca)




#### To Test:

- cd login-workflow
- yarn start:example
- Added console to see email in `forgotPassword` in AuthUIActions
- Open drawer
- Press Auth Provider
- Check the console



#### Any specific feedback you are looking for?

-
